package.json

```diff
@@ -56,7 +57,10 @@
     "prettier": "^1.17.1",
     "sinon": "^7.3.2",
     "ts-node": "^8.2.0",
-    "typescript": "^3.5.1"
+    "typescript": "^3.5.1",
+    "tslint": "^5.16.0",
+    "tslint-no-unused-expression-chai": "^0.1.4",
+    "tslint-plugin-prettier": "^2.0.0"
```

Comment:
Why are these out of order? Were they added with `npm install --save-dev`?

Reply:
They were not, I just added them manually and slipped on the alphabetizing, will fix :)

Reply:
Fixed.